binaries/speed_benchmark_torch.cc

```diff
@@ -77,10 +81,17 @@ std::vector<c10::IValue> create_inputs() {
 
   std::vector<std::string> input_dims_list = split(';', FLAGS_input_dims);
   std::vector<std::string> input_type_list = split(';', FLAGS_input_type);
+  std::vector<std::string> input_memory_format_list =
+      split(';', FLAGS_input_memory_format);
+
   CAFFE_ENFORCE_EQ(
       input_dims_list.size(),
       input_type_list.size(),
       "Input dims and type should have the same number of items.");
+  CAFFE_ENFORCE_EQ(
+      input_dims_list.size(),
+      input_memory_format_list.size(),
+      "Input dims and format should have the same number of items.");
```

Comment:
I'm a little concerned that this will break existing use cases where we're explicitly specifying the dims and types.  If there are more than two inputs, this will cause the benchmark to fail if the memory format isn't specified, right?